src/apps/administration/index.md

```diff
@@ -0,0 +1,73 @@
+<script setup>
+import SwagLanding from "../../components/SwagLanding.vue";
+import SwagLandingCard from "../../components/SwagLandingCard.vue";
+import SwagLandingCardList from "../../components/SwagLandingCardList.vue";
+import * as HeroIcons from "@heroicons/vue/24/solid";
+</script>
+
+<SwagLanding>
+    <template #title>Administration</template>
+    <template #description>
+        <p>Shopware's administration manages all aspects of your store. It outlines all internal processes like product inventory tracking, order processing, customer record management, etc. The Administration component is tightly coupled with Shopware's core, which helps to manage administrative activities.</p>
+        <h1>Capabilities</h1>
+    </template>
+    <template #im>
+        <img src="/landing/apps/admin.PNG"/>
+    </template>
+    <template #exposed2>
+        <SwagLandingCardList>
+            <template #title>
+                Explore the possibilities you can use to extens your Administration component
+            </template>
+            <template #cards>
+                <SwagLandingCard link="/docs/guides/plugins/apps/administration/add-custom-modules">
+                    <template #title>Define features</template>
+                    <template #sub>Customize platform - Define catalog, order settings, etc. with modules.</template>
+                    <template #icon>
+                      <div>
+                        <HeroIcons.TagIcon class="h-10 w-10 text-blue-500" />
```

Comment:
It would make sense to pass the icon as a prop - `SwagLandingCard icon="TagIcon"` and set default style in the component directly so we can change the style in one place (instead of having it scattered across markdown files).